src/pages/edge/adobe-journey-optimizer/content-card-ui/iOS/tutorial/listening-content-card-events.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Listening to Content Card Events
+description: Tutorial on how to listen to content card events in your application.
+keywords:
+- Adobe Journey Optimizer
+- Guide
+- Content Card
+- Messaging
+- Customizing UI
+- Card Templates
+- Content Card Templates
+- ContentCareUI
+- Dismiss
+- Interact
+- Display
+---
+
+# Listening to Content Card Events
+
+This tutorial explains how to listen to content card events in your application.
+
+## Overview
+
+The Messaging extension provides a way to listen to events from content cards displayed in your application. The following functions can be implemented in conformance with the [ContentCardUIEventListening](../public-classes/contentcarduieventlistening.md) protocol:
+
+- `onDisplay`
+- `onDismiss`
+- `onInteract`
+
+## Implement ContentCardEventListening
+
+Complete the following steps to hear content card events:
```

Comment:
I'm not sure if hear here is the right word?

Reply:
Changed to listen